server/controllers/lists/update.js

```diff
@@ -0,0 +1,19 @@
+const { updateAttributes } = require('controllers/lists/lib/lists')
+
+const sanitization = {
+  list: {},
```

Comment:
The other controllers use the `id` attribute name, which makes that one inconsistent, no? That inconsistency seems to have been imported from shelves, maybe it could be made consistent there too, but that would be a breaking change that would require a client update, so maybe in another branch to not complexify this PR's release process

Reply:
joys of copy/pasting

!fixup implement basic CRUD